internal/api/design/convert.go

```diff
@@ -0,0 +1,12 @@
+package design
+
+// sliceToAny copies the elements of a slice with a specific type (e.g.
+// "[]string") to a slice with type []any. See:
+// https://go.dev/doc/faq#convert_slice_of_interface.
+func sliceToAny[T any](s []T) []any {
+	a := make([]any, len(s))
+	for i, v := range s {
+		a[i] = v
+	}
+	return a
+}
```

Comment:
There is an `EnumNameInterfaces()` method available in the enum template already ([here](https://github.com/artefactual-sdps/enduro/blob/main/hack/make/enums.tmpl#L6-L13)), you should be able to use that function directly.

Reply:
Ah, nice! Thanks. :)